### PR TITLE
User/bulelani/cb 1659

### PR DESCRIPTION
**Pull request by bxaia**
Added dropdown to select video feed source on the gui,

### CODE REVIEW

File: app/video/video.js [hunk 1]
```diff
@@ -267,9 +267,9 @@
         var unbindUpdate = $rootScope.$on('sensorsServerUpdateMessage', function (event, sensor) {
             var sensorName = sensor.name.split(':')[1].replace('anc_', '');
             vm.sensorValues[sensorName] = sensor.value;
-            if (sensorName === 'vds_focus_position') {
+            if (sensorName === vm.imageSource.name + '_focus_position') {
```
Comment (ofaletse) [inline app/video/video.js]: vm.imageSource.name will return vdslos_source

**Comment by lvdheever** [comment]
@ofaletse You happy with this one? Can it be merged?

**Comment by ofaletse** [comment]
@lvdheever There is still one last change to be made, should be merged before 9h30 today.

**Comment by ofaletse** [comment]
It looks okay for me.

**Pull request merged**